web/src/compose_validate.ts

```diff
@@ -264,6 +286,9 @@ export function warn_if_topic_resolved(topic_changed) {
     }
 
     const stream_id = compose_state.stream_id();
+    if (!stream_id) {
+        return;
+    }
```

Comment:
This is a weird place to do this check; see https://github.com/zulip/zulip/pull/29268 for a commit fixing up this part of the code that you should be able to rebase onto.